.Rbuildignore

```diff
@@ -15,6 +15,7 @@
 ^NEWS\.0\.md$
 ^README\.md$
 ^_pkgdown\.yml$
+^src/Makevars$
```

Comment:
We need to ignore the generated Makevars in both git and the built package, we generate it from `src/Makevars.in` in the `configure` script.